From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Thu, 12 Jul 2018 11:00:45 -0400
Subject: [PATCH] Don't build the fdt command.

Don't build the fdt command separately but *do* build it into the arm64 kernel.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 grub-core/Makefile.core.def | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/grub-core/Makefile.core.def b/grub-core/Makefile.core.def
index 9039a6f73a3..aabe53262f0 100644
--- a/grub-core/Makefile.core.def
+++ b/grub-core/Makefile.core.def
@@ -246,6 +246,7 @@ kernel = {
 
   arm64_efi = kern/arm64/efi/init.c;
   arm64_efi = kern/efi/fdt.c;
+  arm64_efi = loader/efi/fdt.c;
 
   i386_pc = kern/i386/pc/init.c;
   i386_pc = kern/i386/pc/mmap.c;
@@ -1729,12 +1730,14 @@ module = {
   efi = loader/efi/linux.c;
 };
 
+/*
 module = {
   name = fdt;
   arm64 = loader/efi/fdt.c;
   common = lib/fdt.c;
   enable = fdt;
 };
+*/
 
 module = {
   name = xnu;
